akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/BehaviorTestKitImpl.scala

```diff
@@ -164,3 +169,42 @@ private[akka] final class BehaviorTestKitImpl[T](_path: ActorPath, _initialBehav
 
   override def receptionistInbox(): TestInboxImpl[Receptionist.Command] = context.system.receptionistInbox
 }
+
+private[akka] object BehaviorTestKitImpl {
+  object Interceptor extends BehaviorInterceptor[Any, Any]() {
+
+    /**
+     * Intercept a message sent to the running actor. Pass the message on to the next behavior
+     * in the stack by passing it to `target.apply`, return `Behaviors.same` without invoking `target`
+     * to filter out the message.
+     *
+     * @return The behavior for next message or signal
+     */
```

Comment:
This is just a copy of the documentation on the generic `BehaviorInterceptor.aroundReceive` method, not documentation on this particular implementation. I think it would be clearer to either remove it, or document what this particular implementation does: something like "adapts messages when there is a MessageAdapter registered for it, otherwise passes it to the target unchanged" ?